editor/src/components/editor/canvas-toolbar.spec.browser2.tsx

```diff
@@ -520,6 +521,77 @@ export var storyboard = (
 `)
   })
 
+  it('can insert two elements one after the other', async () => {
+    const editor = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`<div
+    style={{
+      backgroundColor: '#aaaaaa33',
+      position: 'absolute',
+      left: 57,
+      top: 168,
+      width: 247,
+      height: 402,
+    }}
+    data-uid='container'
+  >
+    <div data-uid='a3d' />
+  </div>`),
+      'await-first-dom-report',
+    )
+
+    await selectComponentsForTest(editor, [
+      EP.fromString(`${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:container`),
+    ])
+
+    await expectNoIrrecoverableErrors(editor, async () => {
+      FOR_TESTS_setNextGeneratedUids(['reserved', 'new-img'])
+      await insertViaAddElementPopup(editor, 'img')
+      await selectComponentsForTest(editor, [
+        EP.fromString(`${BakedInStoryboardUID}/${TestSceneUID}/${TestAppUID}:container`),
+      ])
+      FOR_TESTS_setNextGeneratedUids(['reserved', 'new-img-2'])
```

Comment:
is this a fair test? in order to trigger the fixElement shouldn't you set this to `new-img` too? or is this `setNextGeneratedUids` here to feed the fixElement's UID generation?